### ISSUE TITLE
Error in _prob2cornerflow() function

### ISSUE THREAD

**Issue by JasonGUTU** [ucbdrive/hd3#39]
https://github.com/ucbdrive/hd3/blob/84792e27eec81ed27671018c231538e6de8cd6be/models/hd3_ops.py#L207

In this line, the indice is of type long int, and the d is also an int. however, the results of / maybe a float type number, resulting in an error in the following calculation.

Can you help me with this line? What is it about?

**Comment by dbm9741** [comment]
I have the same issue.... If you try to run it using Pytorch 1.0 the problem goes away.

I was able to fix the problem by replacing 
https://github.com/ucbdrive/hd3/blob/84792e27eec81ed27671018c231538e6de8cd6be/models/hd3_ops.py#L208

with indice += torch.div(indice, d-1, rounding_mode='trunc')

**Comment by josephinemonica** [comment]
@dbm9741 Is the rounding_mode supposed to be 'trunc' or something else e.g., 'floor'?

**Comment by dbm9741** [comment]
@josephinemonica I don't think it matters as the indices can only be positive number and floor and trunc will give you the same result for positive number.
